Design rules/Policy Design Rules.md

```diff
@@ -0,0 +1,70 @@
+
+Het proces voor het wijzigen, verwijderen van een bestaande of opvoeren van een nieuwe Design Rule of Extension is als volgt:
+
+1. Er wordt in de API-Kennisbank repository een issue ingediend met het label 'Design Rule'. 
+2. Indien het om een voorstel voor een nieuwe Design Rule gaat wordt deze zo duidelijk mogelijk beschreven, zo mogelijk voorzien van voorbeelden en ratio;
+3. Indien het om een voorstel tot wijziging gaat wordt beschreven welke Design Rule moet worden gewijzigd, wat die wijziging inhoudt en wat daarvoor de ratio is;
+4. Betreft het een verwijderingsvoorstel dan wordt beschreven welke Design Rule het betreft en wat de reden is van het verzoek;
+5. Vervolgens wordt er in een eerste reactie een lijstje met initialen van de medewerkers van het Kenniscentrum Standaarden op deze wijze geplaatst:
+
+> JB: <br/>
+> HK: <br/>
+> RM: <br/>
+> MV: <br/>
+> GJ: <br/>
+> JBi: <br/>
+
+6. Het issue wordt assigned aan alle medewerkers van het Kenniscentrum Standaarden;
+7. Elke medewerker geeft vervolgens z.s.m. in de lijst met initialen achter zijn initialen aan hoe hij over het issue denkt ('Overnemen', 'Bespreken', 'Geen mening', 'Toelichting gewenst). Daarnaast kan elke medewerker zijn mening ook nog toelichten in een aparte post. Evt. kan daarin ook worden aangegeven dat het betreffende issue alleen als Extension moet worden opgenomen.
+8. Kan iedereen zich vinden in het issue en de status (Design Rule of Extension) dan wordt het zonder verdere discussie doorgevoerd;
+9. Indien dat niet het geval is dan wordt zo mogelijk de inhoudelijke discussie digitaal gevoerd en beslecht middels github comments. Als dat niet lukt, dan wordt het issue als onderwerp van het teamoverleg Kenniscentrum Standaarden geagendeerd;
+10. In dat overleg wordt besloten of het voorstel wordt gehonoreerd en met welke status (Design Rule of Extension); 
+11. Indien goedgekeurd wordt de Design Rule of de Extension toegevoegd aan [de Design Rules van VNG Realisatie](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/Design%20rules/readme.md).
+12. In de zeldzame gevallen waarin we met een Design Rule structureel afwijken van de landelijke Rest-API-strategie is een "Leg uit"-situatie van toepassing. In de Design Rule moet dan heel goed uitgelegd worden waarop we afwijken en waarom deze afwijking van belang is. In zo'n geval moet ook weer de discussie met de rest van de community worden opgestart om te kijken waar deze verschillen in inzicht in zitten en of we de landelijke strategie dan niet iets losser of juist strakker moeten definiëren.
+
+M.b.t. het wijzigen en verwijderen van Design Rules wordt in de periode tussen het opvoeren van het issue en het besluit in de projecten niet vooruitgelopen op dat besluit. Aanpassingen n.a.v. het honoreren van het voorstel worden dus pas aangebracht nadat het voorstel is gehonoreerd.
```

Comment:
Wat bedoelen we met niet vooruitlopen op? Een project kan echt in de problemen komen als het tegen een fundamenteel issue aanloopt dat we eerder niet onderkend hebben. Dan kan het project toch een explain doen, of mag dat niet?

Reply:
Ik heb deze zin zo aangepast dat er wel vooruitgelopen mag worden op een besluit maar dat in dat geval wel uitleg verschaft moet worden.